test/font/fontutils.js

```diff
@@ -65,24 +65,26 @@ function encodeFontData(data) {
   return buffer;
 }
 
-function ttx(data, callback) {
-  const xhr = new XMLHttpRequest();
-  xhr.open("POST", "/ttx");
+function ttx(data) {
+  return new Promise((resolve, reject) => {
```

Comment:
Can't we just make this function `async` instead so we don't need to wrap it in a `Promise` manually?

Reply:
Unfortunately I cannot see how that'd work, given that `xhr.onreadystatechange` is a *callback* function that we cannot simply `await` :-(

We could *maybe* re-factor this function to use `fetch` instead, assuming this isn't using e.g. a `file:`-URL, but that's probably better left for a follow-up.

Reply:
It seems that using `fetch` instead worked just fine, so I've pushed a separate commit that fixes this :-)